Backpack/src/main/java/net/skyscanner/backpack/calendar2/data/CalendarMonth.kt

```diff
@@ -81,8 +81,10 @@ internal inline fun CalendarMonth(
 // see https://github.com/ThreeTen/threetenbp/issues/55
 @Suppress("DEPRECATION")
 internal fun MonthTitle(yearMonth: YearMonth, formatter: SimpleDateFormat, locale: Locale): String {
-  val date = Date()
-  date.year = yearMonth.year - 1900
-  date.month = yearMonth.monthValue - 1
+  val date = Date(
+    yearMonth.year - 1900,
+    yearMonth.monthValue - 1,
+    2 // Lock the formatter date to 2nd, so it accounts for -12h tz and the month is predictable
```

Comment:
I'm not sure if that's the intended behaviour as this will modify the day in production code to a fixed one

Reply:
MonthTitle()'s purpose is to transform a year and a month to a title for the month, the day is irrelevant. This particular Date() value is only used so the month formatter could work. However, it wasn't working correctly because when creating a new Date(), it would take the current date as default, e.g. 2021-08-30, then we were changing the year and month without updating the day. This led to having a date of 2021-02-30 which is technically invalid and the formatter interpreted this as March rather than February.

So given the scoped use of this Date() object it is ok to use a static day.

Reply:
aaaah got it